data/wanderers middle.txt

```diff
@@ -1094,11 +1093,10 @@ mission "Wanderers: Mind 6"
 
 event "wanderers: kor mereti transformation"
 	system Chimitarp
-		fleet "Small Kor Mereti" 200
-		fleet "Large Kor Mereti" 400
+		add fleet "Small Kor Mereti" 250
```

Comment:
These have a larger period because there is an existing definition of small/large Kor Mereti fleets in Chimitarp, and the combined result has average period 200 / 400.

If we change how add/remove works for fleet periods ( #2742 ), this should be revisited as well.